src/export/digitalPlanning/model.test.ts

```diff
@@ -35,7 +35,7 @@ const mockMetadataForSession = (
 ): SessionMetadata => ({
   id: "c06eebb7-6201-4bc0-9fe7-ec5d7a1c0797",
   createdAt: "2023-01-01 00:00:00",
-  submittedAt: "2023-01-02 00:00:00",
+  submittedAt: "2023-01-02T12:17:33.199914+00:00",
```

Comment:
This does match the format of the `lowcal_session.submitted_at` column, I've not just changes this to make the tests pass!

<img width="784" height="639" alt="Image" src="https://github.com/user-attachments/assets/61a9f999-f739-4d02-875e-355c68fc2cf7" />